packages/vscode-extension/src/common/Project.ts

```diff
@@ -55,6 +55,14 @@ export const StartupStageWeight = [
   { StartupMessage: StartupMessage.AttachingDebugger, weight: 1 },
 ];
 
+export type ReloadAction =
+  | "rebuild"
+  | "reboot"
+  | "reinstall"
+  | "restartProcess"
+  | "reloadJs"
+  | "hotReload";
```

Comment:
Can we have some small docs for the different actions in comments?
For example:
"rebuild" – performs a clean build of the app and clears metro cache
"reboot" – ???